ddtrace/llmobs/_evaluators/ragas/faithfulness.py

```diff
@@ -106,23 +122,31 @@ def __init__(self, llmobs_service):
     def run_and_submit_evaluation(self, span_event: dict):
         if not span_event:
             return
-        score_result = self.evaluate(span_event)
-        if score_result is not None:
+        score_result_or_failure = self.evaluate(span_event)
+        telemetry_writer.add_count_metric(
+            "llmobs",
+            "evaluators.{}.run".format(self.LABEL),
```

Comment:
```suggestion
            "evaluators.run",
```

and tag with the label/evaluator type